### DISCUSSION TITLE
Welcome to UXsim Discussions!

### DISCUSSION THREAD

**Discussion by toruseo** [toruseo/UXsim#54] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EwoutH** [comment]
Thanks for opening the discussions!

I'm Ewout, a Dutch [EPA](https://www.tudelft.nl/onderwijs/opleidingen/masters/engineering-and-policy-analysis/msc-engineering-and-policy-analysis) student at the TBM faculty of the TU Delft. I'm currently working on my masters thesis, investigating the effect self-driving cars (in the form of robotaxis) might have on urban centers, not only in respects to traffic, but also noise and pollution. This also includes looking at modal shift and induced demand.

Furthermore, I'm a core developer and maintainer of two Python libraries:

- [Mesa](https://github.com/projectmesa/mesa), an agent-based modelling library
- The [EMAworkbench](https://github.com/quaquel/EMAworkbench) for deep uncertainty analysis and robust decision making
